pkg/cmd/openshift-apiserver/openshiftapiserver/openshift_apiserver.go

```diff
@@ -545,6 +569,12 @@ func (c completedConfig) New(delegationTarget genericapiserver.DelegationTarget,
 		return nil, err
 	}
 
+	// add aggregator last
```

Comment:
why last? We don't add an apiserver anymore. This just adds a nother controller. Maybe rename it to  `withOpenAPIAggregationController`